extensions/sql-migration/src/api/azure.ts

```diff
@@ -654,17 +655,18 @@ export async function validateIrSqlDatabaseMigrationSettings(
 			userName: migration._sqlServerUsername,
 			password: migration._sqlServerPassword,
 			authentication: migration._authenticationType,
+			encryptConnection: encryptConnection,
 			trustServerCertificate: trustServerCertificate,
-			// encryptConnection: true,
 		},
 		targetSqlConnection: {
 			testConnectivity: testTargetConnectivity,
 			dataSource: targetDatabaseServer.properties.fullyQualifiedDomainName,
 			userName: migration._targetUserName,
 			password: migration._targetPassword,
+			authentication: MigrationSourceAuthenticationType.Sql,
+			// when connecting to a target Azure SQL DB, use true/false
```

Comment:
IR validation for SQL DB scenario - verify that true/true is correct for the target

Reply:
cc @siyangMicrosoft

Reply:
https://learn.microsoft.com/en-us/azure/azure-sql/database/connect-query-content-reference-guide?view=azuresql#tls-considerations-for-database-connectivity

recommendation is 'true' always.

Reply:
yeah, Encrypt to "true" and TrustServerCertification to "false" should always work for MI and Azure SQL DB

Reply:
We should only /always connect to Azure targets using encrypt on and trust off.

Reply:
@brian-harris that doesn't work for SQL VM as the target platform

Reply:
resolving this thread in particular, does seem true/false for DB is correct